appveyor.yml

```diff
@@ -25,33 +25,63 @@ cache:
 
 environment:
   global:
+    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+    # /E:ON and /V:ON options are not enabled in the batch script intepreter
+    # See: http://stackoverflow.com/a/13751649/163740
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\bin\\admin\\run_with_env.cmd"
+
     TRIAL_REPORTER: text
 
   # Only use pre-installed Python versions, which Appveyor may upgrade to
   # a later point release.
   # See: http://www.appveyor.com/docs/installed-software#python
   matrix:
-    - PYTHON: C:\Python27-x64
+    - PYTHON_HOME: C:\\Python27-x64
+      PYTHON_VERSION: "2.7"
+      PYTHON_ARCH: "64"
       TOXENV: py27-coverage-windows,codecov-publish
+    #- PYTHON_HOME: "C:\\PYTHON35-x64"
+    #  PYTHON_VERSION: "3.5"
+    #  PYTHON_ARCH: "64"
+    #  TOXENV: 'py35-tests-windows'
 
 
 install:
+  # Install OpenSSH through chocolatey
```

Comment:
If it's because OpenSSH includes a libssl: pyOpenSSL should be grabbing Cryptography's Windows wheel; we should NOT be building our own cryptography binaries in CI.